### PR TITLE
[RENTAL] Added realization of rental service and controller

### PR DESCRIPTION
**Pull request by OlegTuryk**

### CODE REVIEW

File: src/main/java/ua/mate/team3/carsharingapp/repository/RentalRepository.java [hunk 1]
```diff
@@ -1,9 +1,19 @@
...
+import java.util.List;
+import org.springframework.data.domain.Pageable;
 import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
 import org.springframework.stereotype.Repository;
 import ua.mate.team3.carsharingapp.model.Rental;
 
 @Repository
 public interface RentalRepository extends JpaRepository<Rental, Long> {
+    @Query("SELECT r FROM Rental r "
```
Comment (padu12) [inline src/main/java/ua/mate/team3/carsharingapp/repository/RentalRepository.java]:
```suggestion
    @Query("FROM Rental r "
```

**Review by padu12 — CHANGES_REQUESTED** [review]
Nice but a minor improvement left

**Review by Zel1oy — APPROVED** [review]

File: src/main/java/ua/mate/team3/carsharingapp/model/Rental.java [hunk 2]
```diff
@@ -36,14 +36,14 @@ public class Rental {
     private LocalDateTime actualReturnDate;
 
     @NotNull
-    @OneToOne(fetch = FetchType.LAZY)
+    @ManyToOne(fetch = FetchType.LAZY)
```
Comment (Zel1oy) [inline src/main/java/ua/mate/team3/carsharingapp/model/Rental.java]: Change ManyToOne to OneToOne

**Review by Zel1oy — CHANGES_REQUESTED** [review]
1 Rental = 1 Car

**Review by padu12 — APPROVED** [review]

**Pull request merged**